plugins/inputs/marklogic/README.md

```diff
@@ -0,0 +1,50 @@
+# Marklogic Plugin
+
+The Marklogic Telegraf plugin gathers status metrics from one or more host in the Marklogic cluster.
```

Comment:
Change the case throughout for `Marklogic` to their preferred case - `MarkLogic`